ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -1107,5 +1110,19 @@
     "EndButtonLabel": "Save and continue",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documentsv2/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documentsv2/submitted"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "manageCaseTTL",
+    "Name": "Manage Case TTL",
+    "Description": "Manage Case TTL",
+    "DisplayOrder": 15,
+    "PreConditionState(s)": "Submitted;GATEKEEPING_LISTING;Gatekeeping;PREPARE_FOR_HEARING;FINAL_HEARING;CLOSED",
```

Comment:
Just wondering why you didn't use "*" ?

Reply:
Wasn't sure if we would change this to only be available in certain states as its not specified but ill ask